src/containerapp/azext_containerapp/containerapp_env_decorator.py

```diff
@@ -111,6 +111,9 @@ def get_argument_min_nodes(self):
     def get_argument_max_nodes(self):
         return self.get_param("max_nodes")
 
+    def get_argument_infrastructure_resource_group(self):
+        return self.get_param("infrastructure_resource_group")
```

Comment:
Please move this `get_argument_infrastructure_resource_group` to `ContainerappEnvPreviewCreateDecorator`.
Because `ContainerappEnvCreateDecorator` will be remove, and we will import `ContainerappEnvCreateDecorator` from core: https://github.com/Azure/azure-cli/pull/27078

Reply:
Moved, thank you!